src/framework/fieldMessage/_index.scss

```diff
@@ -1,6 +1,6 @@
 
 $fieldMessage_borderColor: #e6e6ae;
```

Comment:
Agree, created a ticket for cleaning up all colors: https://jira.smaato.net/browse/SDX-1221, so out of scope.